### PR TITLE
Imports actually work now

### PR DESCRIPTION
**Pull request by imbstack**
Each of these changes is relatively self-contained in a commit.

- Allow configuring bq tables
- Stop modifying prod data as if it is test data
- Initial import followups
- Allow loading csvs in prod
- Give up on validating emails for now
- Make debugging imports a bit easier
- Handle duplicates in ingestion
- Move csv modifiers into helper
- Add scripts for modifying csvs

In particular, I'd say checking that the removal of EmailStr makes sense. Much of this is just dumping in semi-horrible scripts that I used to make the imports work. This is just for later use if we need them. I make no promises about them not being TODO-riddled junk.

### CODE REVIEW

**Review by cvalaas — APPROVED** [review]

File: ctms/app.py
Comment (jwhitlock) [inline ctms/app.py:127]: I think we talked about this in other channels, but not in repo.

There are emails that fail pydantic / python-email-validator, such as `foo.@bar.com`. but are in the Salesforce / BigQuery list. I think the validator is correct, and these are invalid emails, potentially undeliverable, and including them in the our systems reduces the usefulness of the data.

There are also records with no emails, which seems even less useful than a invalid, perhaps mistyped email.

I think the right thing is to relax the constraints, like we're doing here, to get over the hump of the SFDC → Acoustic + CTMS transition. We also need an issue for the follow-on work of classifying the suspicious emails and blank values, and tightening up the acceptance of new data.

My hope was that we could be strict with new data and loose with existing data, but that is non-trivial work that has to be sacrificed to the deadline, and this is the right approach.

File: ctms/bin/prepare_csvs.py
Comment (jwhitlock) [inline ctms/bin/prepare_csvs.py:3]: Nit: I was surprised that the linter was OK with this style. I'd expect:
```suggestion
"""
Given a dump of csvs from bigquery, prepare it for being imported
into postgres with COPY.
"""
```

File: ctms/bin/report_duplicates.py
Comment (jwhitlock) [inline ctms/bin/report_duplicates.py:40]: Nit: move `87339348` into a variable

File: tests/unit/test_bin_load_sample_csv.py
Comment (jwhitlock) [inline tests/unit/test_bin_load_sample_csv.py:30]: Nit: Can we put some data in the test duplicates, to get some coverage on the duplicate processor, even if the duplicate data isn't used in tests 

*I can't comment on the empty files, so I'm putting it here*

**Review by jwhitlock — APPROVED** [review]
Thanks @imbstack, this is a ton of work! I can see some follow-on work (converting print to logging, future email validation, etc.) but I think we should merge and deal with that later.

There's a couple of failing tests, and black wants to make some changes, and linting is broken for me. I'm seeing issues that are probably related to my recent commits adding sentry and test linting. I'm not sure the best way to proceed - collaboration on figuring out the problem, or merge and then let me fix it. I'm approving so you can pick the second tactic.

The test failure is from processing the newsletter loop. The input is a list of dictionaries, and ingest fails to call ``.dict()`` on a dictionary.

**Comment by jwhitlock** [comment]
I dug a little into my `pylint` issues. They go away if I remove `sentry_sdk.init` from `ctms/app.py`. They appear in branch for #145 and in the `main` branch. I'm surprised that linting doesn't run in CI.

I think the quickest fix is to revert the changes to `pyproject.yml` and `scripts/lint.sh` from https://github.com/mozilla-it/ctms-api/commit/e7e7d708f0ea55f4cecd2c2cf51a56c6d5bddf3a, so that tests are no longer linted. I think the code changes in the tests to fix linting are OK. I could do this in a separate PR, before or after this merges, or it could be a commit in this PR.

For the long-term fix, I'll need to re-arrange how ``sentry_sdk.init`` is called. It may be enough to move it into ``startup_event``. It may be necessary to call it from any scripts as well.

File: tests/unit/test_bin_load_sample_csv.py
Comment (imbstack) [reply tests/unit/test_bin_load_sample_csv.py:30]: Tracked in #148

File: ctms/app.py
Comment (imbstack) [reply ctms/app.py:127]: Yeah, agreed. I think I feel much better now that this data is in the db successfully. We can start processing it shortly.

**Pull request merged**
